### DISCUSSION TITLE
vue-chartjs updated with Vue 3 and Chart.js 3 support

### DISCUSSION THREAD

**Discussion by victorgarciaesgi** [victorgarciaesgi/vue-chart-3#118] [Announcements]
I can't be as active as them, and their solution is mostly similar so I think it's better to switch to `vue-chartjs` if you want better support. It was a great ride to write this library though and I learnt a lot so thanks for your support! :D

**Comment by jacekkarczmarczyk** [comment]
Thanks for the info and for your work!

**Comment by dargmuesli** [comment]
Notice: `vue-chartjs` does not yet support reactive options! https://github.com/apertureless/vue-chartjs/issues/805

**Comment by thabarbados** [comment]
Hey folks. I'm a co-maintainer of the [vue-chartjs](https://github.com/apertureless/vue-chartjs) library. 

What do you need to migrate apps to vue-chartjs? Maybe some features are required, a migration guide, etc.?
We will add reactive options shortly.

**Reply by dargmuesli** [reply]
Actually the migration was pretty straight forward for me: https://github.com/maevsi/maevsi/pull/586/files
- uninstalled `vue-chart-3`, installed `vue-chartjs`
- renamed `PieChart` to `Pie`
- change import from `vue-chart-3` to `vue-chartjs/legacy`

Reactive options would be great! Otherwise I only noticed that the width and height setters work differently, not sure how though. Fixed that with kind of a workaround, there might be better ways.

**Reply by victorgarciaesgi** [reply]
Hi! When I have time I will help bring reactive options to `vue-chartjs`. I managed to let my company give me some time to do open source stuff (even tho they are not using Chart.js, they're cool aha).
I would also like to explore the composables I experimented on vue-chart-3, but I don't have many feedbacks on this

**Reply by thabarbados** [reply]
PR to add reactive options will be ready tomorrow.

**Reply by victorgarciaesgi** [reply]
Wow sorry I didn't kept track on this! 😅
